### PR TITLE
feat: hide some functionalities behind feature flags

### PR DESCRIPTION
**Pull request by Iipin**
did i test it? nah

### CODE REVIEW

**Comment by m4tx** [comment]
chad

**Comment by m4tx** [comment]
I'm wondering... Can we make `clap` read the config from a file? I think I'd prefer to have a config file rather than add flags to CLI. Ideally, we could have both.

**Comment by m4tx** [comment]
On the other hand, env file could be treated as, well... config file. So I guess the current state is fine too.

**Comment by Iipin** [comment]
Yeah, it also reads environment (just like the previous version). We could also add e.g. https://crates.io/crates/dotenvy which would essentially make `.env` a config file :trollface:.

File: src/slash_commands/mod.rs [hunk 1]
```diff
@@ -35,18 +36,19 @@ pub struct SlashCommands {
...
-    fn get_slash_commands() -> Vec<Box<dyn SlashCommand>> {
-        vec![
-            Box::new(ChomboCommand::new()),
-            Box::new(HandCommand::new()),
-            Box::new(PastaCommand::new()),
-        ]
+    fn get_slash_commands(args: &Arguments) -> Vec<Box<dyn SlashCommand>> {
+        let mut ret: Vec<Box<dyn SlashCommand>> =
+            vec![Box::new(HandCommand::new()), Box::new(PastaCommand::new())];
+        if args.feature_kcc3 {
```
Comment (m4tx) [inline src/slash_commands/mod.rs:48]: Maybe we should allow disabling `/pasta` command as well?

Comment (Iipin) [reply src/slash_commands/mod.rs:48]: heresy

**Review by m4tx — APPROVED** [review]

**Pull request merged**